test_scripts/AppServices/RPCPassThrough/009_UnknownRPC_Consumer_UNSUPPORTED_REQUEST.lua

```diff
@@ -0,0 +1,126 @@
+---------------------------------------------------------------------------------------------------
```

Comment:
This is an outdated version of one of my tests: https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/2188/files#diff-a0d996330af833ecde62a2071206c093

Reply:
I see how it is different, using `allow_unknown_rpc_passthrough` as false. Still, I might suggest using the latest version of https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/2188/files#diff-aecc1389162e76ba570ab26084cb9d94 as a base for this.